src/screens/Deactivated.tsx

```diff
@@ -104,15 +104,15 @@ export function Deactivated() {
   }, [_, agent, setPending, setError, queryClient])
 
   return (
-    <View style={[a.h_full_vh, a.flex_1, t.atoms.bg]}>
+    <View style={[a.util_screen_outer, a.flex_1, t.atoms.bg]}>
       <ScrollView
         style={[a.h_full, a.w_full]}
         contentContainerStyle={{borderWidth: 0}}>
         <View
           style={[
             a.px_2xl,
             {
-              paddingTop: isWeb ? 64 : insets.top,
+              paddingTop: isWeb ? 64 : insets.top + 16,
```

Comment:
Nit: feel like we could use a lil more padding here on native, and for consistency, could do like `a.pt_2xl.paddingTop` or `tokens.space._2xl`

Reply:
Note: Android looks fine, iOS felt a little close to me but clearly it's gonna vary a bit, so not a huge deal

Reply:
bumped to `xl`, I think `2xl` might be a bit much even on iOS